snippets/csharp/VS_Snippets_CLR/conceptual.basicio.textfiles/cs/source.cs

```diff
@@ -8,30 +8,29 @@ class WriteTextFiles
     static void Main(string[] args)
     {
 
-        // Example 1: shows how to write synchronously
+        // Example 1: Shows how to write synchronously
         // to a text file using StreamWriter
         // line by line
         WriteLineByLine();
 
-        // Example 2: Appending text to
+        // Example 2: Shows how to append text to
         // an existing file using StreamWriter
         AppendTextSW();
 
-        // Example 3: shows how to write a 
+        // Example 3: Shows how to write  
         // a simple string asynchronously
         // to a text file using StreamWriter
-        string text = "This is a sentence.";
-        WriteTextAsync(text);
+        WriteTextAsync();
 
-        // Example 4: shows how to write synchronously
+        // Example 4: Shows how to write synchronously
         // to a text file using File and then
-        // adding additional lines 
+        // add additional lines 
         WriteFile();
     }
 
+    // Example 1: Write line by line to a new text file with StreamWriter
     static void WriteLineByLine()
     {
-        // <SnippetWriteLine>
```

Comment:
These markers should stay.  They indicate blocks of code that are included in the corresponding article.

This comment applies to all the following comments that contain the `<Snippet*>` and `</Snippet*>` text.